spec/requests/bot_spec.rb

```diff
@@ -14,7 +15,7 @@
 
     it 'is not redirected and can access secure pages' do
       expect(response).not_to redirect_to new_user_session_path
-      expect(response.body).to include bot_email
+      expect(response.body).to include bot_name
```

Comment:
What makes this necessary?

Reply:
@peterdavidhamilton Email is no longer displayed on account page, so using the name to identify the user in this case instead